packages/create-cloudflare/src/helpers/packages.ts

```diff
@@ -88,9 +89,14 @@ export async function getLatestPackageVersion(packageSpecifier: string) {
  */
 export const installWrangler = async () => {
 	const { npm } = detectPackageManager();
+	const wrangler = process.env.CI
```

Comment:
Why make this CI only?

What if you make the WRANGLER env var more specific (e.g. C3_WRANGLER_OVERRIDE or something) and then use that as the flag to override the `wrangler@latest`?

Then we can use this for local testing too.